crates/sparrow-api/src/kaskada/v1alpha/plan_impl.rs

```diff
@@ -423,6 +425,7 @@ impl std::fmt::Display for super::Literal {
                 write!(f, "interval_months:{i}")
             }
             Some(literal::Literal::Utf8(ref str)) => write!(f, "\\\"{str}\\\""),
```

Comment:
This `ref str` syntax should be unnecessary (it's mostly obsolute). I'd be curious to know why we need it.

Reply:
Removed ref syntax